config/crd/bases/10-crd-druid.gardener.cloud_etcds.yaml

```diff
@@ -284,6 +284,14 @@ spec:
               etcd:
                 description: EtcdConfig defines parameters associated etcd deployed
                 properties:
+                  clientService:
+                    description: ClientService defines specified annotations that should be added to the client service
```

Comment:
The description specifies only annotations. If that is the case then why have another struct for that? If you feel that you can have additional client service config added then you will have to change the description.

Reply:
Yes, you're right
Updated description